Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs

```diff
@@ -162,7 +162,10 @@ public override int GetHashCode()
 
         public void OnPostSceneQuery()
         {
-
+            if (isSelectPressed && IsInteractionEnabled)
```

Comment:
>IsInteractionEnabled [](start = 35, length = 20)

For my own knowledge, what's the reason that only the GGV pointer has the IsInteractionEnabled check?

Reply:
For consistency with the way we raise the down and up events in OnInputDown/Up(). Maybe @julenka can give background on why it is done that way.

Reply:
Just because we don't want to raise pointer down or up events if the pointer is not enabled, meaning in GGVPointer case to not raise events when the object is not active. For other interactions for example hand rays, we don't want to raise events if say the hands are facing the user and the rays should be off